Unable to build react-0.14

I have the following error message
```javascript
ERROR in ./~/react-querystring-router/src/router.js
Module not found: Error: Cannot resolve module 'react-dom' in /cosmos/webpack-boilerplate/react-0.14/node_modules/react-querystring-router/src
```

### Solution

`npm i --save react-dom`

--- a/webpack-boilerplate/react-0.14/README.md
+++ b/webpack-boilerplate/react-0.14/README.md
@@ -16,7 +16,7 @@ Integrating this boilerplate into your codebase should be fairly straightforward
 
 You can see all dependencies inside [`package.json`](package.json), but here's some handy copy-paste commands:
 ```
-npm install --save react@0.14 lodash
+npm install --save react@0.14 react-dom@0.14 lodash
 npm install --save-dev babel-core@5 babel-loader@5 babel-plugin-react-transform@1 express react-component-playground@0.4 react-querystring-router@0.3 react-transform-catch-errors@1 react-transform-hmr@1 redbox-react@1 webpack-dev-middleware webpack-hot-middleware webpack yargs
 
 # Optional, used in this example

--- a/webpack-boilerplate/react-0.14/package.json
+++ b/webpack-boilerplate/react-0.14/package.json
@@ -8,7 +8,8 @@
   },
   "dependencies": {
     "lodash": "^3.10.1",
-    "react": "^0.14.2"
+    "react": "^0.14.2",
+    "react-dom": "^0.14.3"
   },
   "devDependencies": {
     "babel-core": "^5.8.34",

